cli: enable `constellation apply` to create new clusters (#2549)
edgelesssys/constellation: docs/docs/architecture/orchestration.md

The CLI stores state in the local filesystem making the current directory the active workspace.
Multiple clusters require multiple workspaces, hence, multiple directories.
Note that every operation on a cluster always has to be performed from the directory associated with its workspace.
While files from the workspace may be copied to different locations,

> they should'nt be moved or deleted while the Constellation cluster is still being used.

Reviewer: they shouldn't be moved or deleted while the Constellation cluster is still being used.

The Constellation CLI will take care of managing the workspace.

> Only when a cluster was terminated, and you are sure the files are'nt needed anymore, should you remove a workspace.

Reviewer: Only when a cluster was terminated, and you are sure the files aren't needed anymore, should you remove a workspace.

The CLI stores state in the local filesystem making the current directory the active workspace.
Multiple clusters require multiple workspaces, hence, multiple directories.

> Note that every operation on a cluster always has to be performed from the directory associated with its workspace.
> While files from the workspace may be copied to different locations,
> they should'nt be moved or deleted while the Constellation cluster is still being used.
> The Constellation CLI will take care of managing the workspace.
> Only when a cluster was terminated, and you are sure the files are'nt needed anymore, should you remove a workspace.

Reviewer 2: ```suggestion
Note that every operation on a cluster always has to be performed from the directory associated with its workspace.

You may copy files from the workspace to other locations,
but you shouldn't move or delete them while the cluster is still being used.
The Constellation CLI takes care of managing the workspace.
Only when a cluster was terminated, and you are sure the files aren't needed anymore, should you remove a workspace.
```
suggest starting a new paragraph and some rewording
